### ISSUE TITLE
New Eridil Start

### ISSUE THREAD

**Issue by GTink911** [zorkind/Hellion-Rescue-Project#322]
Note: This feature request is likely going to need 0.2.0 revisions to even be feasible.

A TLDR is available in the Additional Context section.

**Is your feature request related to a problem? Please describe.**
Slightly. Most (all?) vanilla starts are focused on Bethyr and the planets farther out than it. I hope to fix this with the start.

**Describe the solution you'd like**
I'm going to break this start down into three main parts: The Story, The Start, and The Unlock.

### The Story

The main story is that your teammate is a SoE spy. Upon receiving orders from command, they have rammed a ship into the station, and set the self destruct sequence. This should also be the basis for the blurb.

### The Start

You start in a derelict module, with a T2 suit - outside a cryopod. You should have some average gear (welding tool, maybe some nanite cores, as if you were performing maintenance) in your inventory. At the start, there are two stations. One is drifting away from your slowly, one you are inside and is attached to the asteroid. In the station your on (attached to asteroid), there's some tiered loot as well as some modules.  The main thing there is a CM which you need to summon a rescue ship with. Make sure some doors are locked and there are some turrets to stop you. Make up a story excuse for not being able to stop the self destruct (Broken panel? Emergency override by a superior? Perhaps the SoE was a pretty high rank). Again, more filler loot. The other station, maybe 100m away from starting position, there is another derelict attached to more modules, as if there was a explosion there - breaking the module you are in into halves. It should be floating away at approx. 15m/s. Nothing notable there, just more loot, maybe some story info. After a 30m-1h, have the asteroid and station explode and despawn, killing any player within about 5km.

### The Unlock

How To Unlock: Make a new side quest named "A Forgotten Tale".  The player discovers a unknown distress signal when they first arrive at Eridil. Upon going to the signal source, (UPDATE: Joaco suggested instead of a signal source, it's a heat signature from the explosion. I like this a lot more, personally) they find the remnants of the exploded asteroid and station. Plenty of resources about, etc. Should emulate the start as if the station had exploded, while still leaving the fate of the inhabitants unknown - they could have either escaped or died. This lets the player imagine this piece of the story how they want, while still accommodating the player either failing or successfully escaping on replays. Once the player interacts with a terminal, they learn a bit about the start in a story friendly manner (Automated ship logs and potential (not stupidly broken like Theona) audio logs?), and gain access to it.

**Describe alternatives you've considered**
Place the start at Hirath instead.

**Additional context**
I really like the idea of starts representing a point in time, which the player then goes into the role of one of the people there. I definitely tried to go for that in the unlock section - first you learn about what happened there and the people in it, then when you play it you actually are the crewmate of the SoE agent. IMO it is a great way to put the player in someone else's shoes.

TLDR Reqular Edition:
Story: Teammate was SoE, damaged station and set self destruct
Start: (Very incomplete! Better to read main section!) Start in asteroid base in pieces. Base will destroy itself and anything in 5km in 30m. Battle to CM, make story excuse why you cant disable self destruct, escape with epicc loot.
How to Unlock: Sidequest, basically find the leftovers of this start and read log to find out what happened.

Ultra Condensed TLDR:
Story: Teammate spy, haha self destruct go brrr
Start: Damaged station, battle turrets (not Battle Turtles!) and escape with loot
How to Unlock: Find signal, interact with terminal and find out about what happened in the start
